### DISCUSSION TITLE
Receive json and print to a file without format

### DISCUSSION THREAD

**Discussion by josiaslg83** [adnanh/webhook#594] [Q&A]
Hi.
I'm trying to receive a json and simple print the receipt to a file, but without success.
I'm trying to use the echo command and, with the instruction received by the url in json, print directly to a file. Is there any way to do this?. All examples just trigger scripts.

**Comment by go2tom42** [comment]
You want to dump the payload to a json file?

Here is part of the hooks.json
```
    "id": "test",
    "execute-command": "/scripts/test.sh",
    "command-working-directory": "/scripts",
    "pass-file-to-command": 
    [
      {
        "source": "raw-request-body",
        "envname": "PFILE"
      }
    ],
```

test.sh contains

```
#!/bin/sh

cat "$PFILE"  >> "./test.json"
```
